contracts/src/CrossChainApplications/GETTING_STARTED.md

```diff
@@ -20,245 +20,277 @@ pragma solidity 0.8.18;
 
-> `allowedRelayerAddresses` is empty in this example, meaning any relayer can try to deliver this cross chain message. Specific relayer addresses can be specified to ensure only those relayers can deliver the message.
-> The `message` must be ABI encoded so that it can be properly decoded on the receiving end.
+> Note: `allowedRelayerAddresses` is empty in this example, meaning any relayer can try to deliver this cross chain message. Specific relayer addresses can be specified to ensure only those relayers can deliver the message.
 
 ```solidity
-emit SendMessage({
-    destinationBlockchainID: destinationBlockchainID,
-    destinationAddress: destinationAddress,
-    feeTokenAddress: feeTokenAddress,
-    feeAmount: adjustedFeeAmount,
-    requiredGasLimit: requiredGasLimit,
-    message: message
-});
-return
-    teleporterMessenger.sendCrossChainMessage(
-        TeleporterMessageInput({
+    function sendMessage(
+        ...
+    ) external returns (bytes32 messageID) {
+        ...
+```
+```solidity
+        emit SendMessage({
             destinationBlockchainID: destinationBlockchainID,
             destinationAddress: destinationAddress,
-            feeInfo: TeleporterFeeInfo({
-                feeTokenAddress: feeTokenAddress,
-                amount: adjustedFeeAmount
-            }),
+            feeTokenAddress: feeTokenAddress,
+            feeAmount: adjustedFeeAmount,
             requiredGasLimit: requiredGasLimit,
-            allowedRelayerAddresses: new address[](0),
-            message: abi.encode(message)
-        })
-    );
+            message: message
+        });
+        return
+            teleporterMessenger.sendCrossChainMessage(
+                TeleporterMessageInput({
+                    destinationBlockchainID: destinationBlockchainID,
+                    destinationAddress: destinationAddress,
+                    feeInfo: TeleporterFeeInfo({
+                        feeTokenAddress: feeTokenAddress,
+                        amount: adjustedFeeAmount
+                    }),
+                    requiredGasLimit: requiredGasLimit,
+                    allowedRelayerAddresses: new address[](0),
+                    message: abi.encode(message)
+                })
+            );
+```
+```solidity
+}
 ```
 
 With the sending side complete, the next step is to implement `ITeleporterReceiver.receiveTeleporterMessage`. The receiver in this example will just receive the arbitrary string data, and check that the message is sent through Teleporter.
 
 ```solidity
-// Receive a new message from another chain.
-function receiveTeleporterMessage(
-    bytes32 sourceBlockchainID,
-    address originSenderAddress,
-    bytes calldata message
-) external {
-    // Only the Teleporter receiver can deliver a message.
-    require(msg.sender == address(teleporterMessenger), "Unauthorized.");
-
-    // do something with message.
-}
+    /**
+     * @dev Receive a new message from another chain.
+     */
```

Comment:
I don't think we need the dev tag on the example code

Reply:
It was already on some things, but not others, so I added it on everything else to be consistent. My only preference is that we be consistent. Would you prefer we skip the natspec entirely in this code?

Reply:
If the goal is to replicate the ExampleCrossChainMessenger, I think it's alright to leave the natspec comments in.

Reply:
I don't really care that much, but does seem like we don't really need it.